pkg/translator/opencensus/oc_to_traces_test.go

```diff
@@ -92,12 +92,9 @@ func TestInitAttributeMapFromOC(t *testing.T) {
 	}
 	attrs = pcommon.NewMap()
 	initAttributeMapFromOC(ocAttrs, attrs)
-	assert.EqualValues(t,
-		pcommon.NewMapFromRaw(
-			map[string]interface{}{
-				"abc": "def",
-			}),
-		attrs)
+	expectedAttr := pcommon.NewMap()
+	expectedAttr.UpsertString("abc", "def")
+	assert.EqualValues(t, expectedAttr, attrs)
```

Comment:
Alternative to this is:

```suggestion
	assert.Equal(t, map[string]interface{}{"abc": "def"}, attrs.AsRaw())
```